<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Chao Dai" />


<title>LeafcutterMD Analyses - GTEx Muscle</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">snaptron</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">LeafcutterMD Analyses - GTEx Muscle</h1>
<h4 class="author">Chao Dai</h4>
<h4 class="date">updated 2022-08-15</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-08-15
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>snaptron/analysis/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed19900924code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(19900924)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed19900924code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(19900924)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomdaiucsnaptrontree5383cf9fe5bff3fdb48485809beb1057cb051ffctargetblank5383cf9a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/daiuc/snaptron/tree/5383cf9fe5bff3fdb48485809beb1057cb051ffc" target="_blank">5383cf9</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomdaiucsnaptrontree5383cf9fe5bff3fdb48485809beb1057cb051ffctargetblank5383cf9a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/daiuc/snaptron/tree/5383cf9fe5bff3fdb48485809beb1057cb051ffc" target="_blank">5383cf9</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/
    Ignored:    .ipynb_checkpoints/
    Ignored:    analysis/.Rhistory
    Ignored:    analysis/.ipynb_checkpoints/
    Ignored:    code/.Rhistory
    Ignored:    code/.ipynb_checkpoints/
    Ignored:    code/.snakemake/
    Ignored:    code/.test/
    Ignored:    code/dag.svg
    Ignored:    code/logs/
    Ignored:    code/playground.R.ipynb
    Ignored:    code/playground.py.ipynb
    Ignored:    code/resources/
    Ignored:    code/results/
    Ignored:    code/slurm_logs/
    Ignored:    code/test.R
    Ignored:    code/test.py
    Ignored:    notebooks/.ipynb_checkpoints/

Untracked files:
    Untracked:  slides/

Unstaged changes:
    Modified:   analysis/index.Rmd
    Deleted:    analysis/leafcutterMD_20220801.Rmd
    Modified:   analysis/leafcutterMD_muscle_20220808.Rmd

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown
(<code>analysis/leafcutterMD_muscle_202208012.Rmd</code>) and HTML
(<code>docs/leafcutterMD_muscle_202208012.html</code>) files. If you’ve
configured a remote Git repository (see <code>?wflow_git_remote</code>),
click on the hyperlinks in the table below to view the files as they
were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/daiuc/snaptron/blob/5383cf9fe5bff3fdb48485809beb1057cb051ffc/analysis/leafcutterMD_muscle_202208012.Rmd" target="_blank">5383cf9</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
<td>
wflow_publish("leafcutterMD_muscle_202208012.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/daiuc/snaptron/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/leafcutterMD_muscle_202208012.html" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/daiuc/snaptron/blob/cc02116dc7a2039b0cac78c10aaad11873654777/analysis/leafcutterMD_muscle_202208012.Rmd" target="_blank">cc02116</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
<td>
wflow_publish("leafcutterMD_muscle_202208012.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/daiuc/snaptron/2bbcc6698e204dd1bcf6b65aef44fa3c1296f2da/docs/leafcutterMD_muscle_202208012.html" target="_blank">2bbcc66</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/daiuc/snaptron/blob/a3d05f2a9932ba150b6b83759b30f9d570b63b8b/analysis/leafcutterMD_muscle_202208012.Rmd" target="_blank">a3d05f2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
<td>
wflow_publish("leafcutterMD_muscle_202208012.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/daiuc/snaptron/df9d86f6884184a426865d02569fbd7220f689b4/docs/leafcutterMD_muscle_202208012.html" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/daiuc/snaptron/blob/cee5962755d4b4f5ca003953961337681021ccea/analysis/leafcutterMD_muscle_202208012.Rmd" target="_blank">cee5962</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
<td>
wflow_publish("leafcutterMD_muscle_202208012.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/daiuc/snaptron/22456a28518e1cf44e6565c1d6816365f0bff13d/docs/leafcutterMD_muscle_202208012.html" target="_blank">22456a2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/daiuc/snaptron/blob/b4bc9bf7624913a93827e827164ef0b30db6d19a/analysis/leafcutterMD_muscle_202208012.Rmd" target="_blank">b4bc9bf</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
<td>
wflow_publish("leafcutterMD_muscle_202208012.Rmd")
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<!-- ---------------------------------------------------- -->
<div id="gtex-muscle" class="section level1 tabset">
<h1 class="tabset">GTEx Muscle</h1>
<div id="group0" class="section level2">
<h2>group0</h2>
<div id="intron-selection" class="section level3">
<h3>Intron selection</h3>
<p>An intron excision is considered outlier only if:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
</ol>
<p>Some samples have many more outlier splicing events
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-9-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-9-1">
Past versions of unnamed-chunk-9-1.png
</button>
</p>
<div id="fig-unnamed-chunk-9-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-9-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/22456a28518e1cf44e6565c1d6816365f0bff13d/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-9-1.png" target="_blank">22456a2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="box-plot-before-quantile-filter" class="section level4">
<h4>Box plot before quantile filter</h4>
<p>There are 14690 outlier introns.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-10-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-10-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/22456a28518e1cf44e6565c1d6816365f0bff13d/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-10-1.png" target="_blank">22456a2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-11-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-11-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="further-filter-outlier-introns-by-comparing-excision-quantiles"
class="section level3">
<h3>Further filter outlier introns by comparing excision quantiles</h3>
<p>By definition, outlier splicing events are ones that occur only in a
few individuals. Here, I further enforced this quantile based
filter:</p>
<ul>
<li>assumes that it occurs in only up to 5% of samples,</li>
<li>thus the excision ratio of the first 95% of samples should be
largely different than that of the last 5% of samples</li>
<li>here, I will only consider the case where the first 95% of samples
have much lower excision ratio, because the complementary set always
exists for a intron cluster. But we only want to select the intron
once.</li>
<li>Specifically, an outlier intron is qualified only if the mean
excision ratio of samples in the (95, 100] percentile is more than 2x sd
away from the 95th percentile, where <code>sd</code> is estimated by
(0,95] percentiles.</li>
</ul>
<p>In addition, for any intron usage of a sample where excision ratio is
1 and observed read counts &lt; 10, read counts and excision ratio is
forced to 0. This helps remove inflated excision ratio of 1 under such
scenarios.</p>
<p>There are 1607 outlier introns after this round of quantile based
filter. The box plot below randomly select 50 out of the outlier introns
(x is sorted by p value).</p>
<p>In summary, the two box plots below plotted some of the outlier
introns that satisfy these criteria:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
<li><strong>AND</strong> quantile filter described above</li>
<li><strong>AND</strong> exclude outliers from extreme samples</li>
</ol>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-13-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-13-1">
Past versions of unnamed-chunk-13-1.png
</button>
</p>
<div id="fig-unnamed-chunk-13-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/2bbcc6698e204dd1bcf6b65aef44fa3c1296f2da/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-13-1.png" target="_blank">2bbcc66</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-13-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/22456a28518e1cf44e6565c1d6816365f0bff13d/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-13-1.png" target="_blank">22456a2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-14-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-14-1">
Past versions of unnamed-chunk-14-1.png
</button>
</p>
<div id="fig-unnamed-chunk-14-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/2bbcc6698e204dd1bcf6b65aef44fa3c1296f2da/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-14-1.png" target="_blank">2bbcc66</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-14-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>outlier per sample after all filters.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-15-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-15-1">
Past versions of unnamed-chunk-15-1.png
</button>
</p>
<div id="fig-unnamed-chunk-15-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-15-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Number of outlier samples per outlier intron
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-16-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-16-1">
Past versions of unnamed-chunk-16-1.png
</button>
</p>
<div id="fig-unnamed-chunk-16-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-16-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="extreme-samples-51070-51390" class="section level3">
<h3>extreme samples: 51070, 51390</h3>
<p>Samples with &gt; 500 outliers</p>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"51070","2":"male","3":"43","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces,  ~12x7mm.  Minimal interstitial fat, excellent specimens.","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Asyphyxiation","12":"Accident","13":"GTEX-WFON-2326-SM-5S2SF"},{"1":"51390","2":"male","3":"27","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces 9x8 & 9x6 mm; no adipose","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"prescription drug overdose","12":"Accident","13":"GTEX-T5JC-0626-SM-5S2RW"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
Samples with less than 100 outliers
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"50123","2":"male","3":"59","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces. 14x9 & 12.5x9mm; Well trimmed, no adipose","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Cardiovascular","12":"Natural","13":"GTEX-XPVG-2526-SM-4B66D"},{"1":"50129","2":"female","3":"52","4":"Muscle","5":"RNA:Total RNA","6":"2  pieces ~ 8x8mm, one with ~30% interstitial fatty aggregate, delineated","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"probable drug overdose","12":"Undetermined","13":"GTEX-WWYW-0526-SM-3NB2W"},{"1":"50153","2":"female","3":"51","4":"Muscle","5":"RNA:Total RNA","6":"Numerous defects. 10x6.5mm","7":"Below Knee-Right","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Presumed Death","11":"Blunt force trauma","12":"Accident","13":"GTEX-OXRK-1826-SM-2HMJE"},{"1":"50156","2":"female","3":"57","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces up to ~11x6mm","7":"Below Knee-Right","8":"Normal","9":"RNA isolation_Trizol Manual (Cell Pellet)","10":"Actual Death","11":"ICH","12":"Natural","13":"GTEX-QCQG-2126-SM-2S1P8"},{"1":"50166","2":"female","3":"68","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, ~5-10% interstitial fat, rep delineated","7":"Gastrocnemius - Left","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"ich","12":"Natural","13":"GTEX-146FR-1726-SM-5Q5DT"},{"1":"50200","2":"female","3":"50","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, includes internal fat (up to 20%) and tendon","7":"Gastrocnemius - Left","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"Cardiovascular","12":"Natural","13":"GTEX-ZZPU-2626-SM-5E45Y"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Notice the outlier sample is not included in the open access dataset
from GTEx. One of the reasons may be that the sample did not pass QC
filters.</p>
<pre><code>character(0)</code></pre>
</div>
</div>
<div id="group1" class="section level2">
<h2>group1</h2>
<div id="intron-selection-1" class="section level3">
<h3>Intron selection</h3>
<p>An intron excision is considered outlier only if:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
</ol>
<p>Some samples have many more outlier splicing events
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-28-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-28-1">
Past versions of unnamed-chunk-28-1.png
</button>
</p>
<div id="fig-unnamed-chunk-28-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-28-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-28-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="box-plot-before-quantile-filter-1" class="section level4">
<h4>Box plot before quantile filter</h4>
<p>There are 4547 outlier introns.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-29-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-29-1">
Past versions of unnamed-chunk-29-1.png
</button>
</p>
<div id="fig-unnamed-chunk-29-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-29-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-29-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/22456a28518e1cf44e6565c1d6816365f0bff13d/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-29-1.png" target="_blank">22456a2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-30-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-30-1">
Past versions of unnamed-chunk-30-1.png
</button>
</p>
<div id="fig-unnamed-chunk-30-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-30-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/2bbcc6698e204dd1bcf6b65aef44fa3c1296f2da/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-30-1.png" target="_blank">2bbcc66</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-30-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/22456a28518e1cf44e6565c1d6816365f0bff13d/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-30-1.png" target="_blank">22456a2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="further-filter-outlier-introns-by-comparing-excision-quantiles-1"
class="section level3">
<h3>Further filter outlier introns by comparing excision quantiles</h3>
<p>By definition, outlier splicing events are ones that occur only in a
few individuals. Here, I further enforced this quantile based
filter:</p>
<ul>
<li>assumes that it occurs in only up to 5% of samples,</li>
<li>thus the excision ratio of the first 95% of samples should be
largely different than that of the last 5% of samples</li>
<li>here, I will only consider the case where the first 95% of samples
have much lower excision ratio, because the complementary set always
exists for a intron cluster. But we only want to select the intron
once.</li>
<li>Specifically, an outlier intron is qualified only if the mean
excision ratio of samples in the (95, 100] percentile is more than 2x sd
away from the 95th percentile, where <code>sd</code> is estimated by
(0,95] percentiles.</li>
</ul>
<p>In addition, for any intron usage of a sample where excision ratio is
1 and observed read counts &lt; 10, read counts and excision ratio is
forced to 0. This helps remove inflated excision ratio of 1 under such
scenarios.</p>
<p>There are 626 outlier introns after this round of quantile based
filter. The box plot below randomly select 50 out of the outlier introns
(x is sorted by p value).</p>
<p>In summary, the two box plots below plotted some of the outlier
introns that satisfy these criteria:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
<li><strong>AND</strong> quantile filter described above</li>
<li><strong>AND</strong> exclude outliers from extreme samples</li>
</ol>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-32-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-32-1">
Past versions of unnamed-chunk-32-1.png
</button>
</p>
<div id="fig-unnamed-chunk-32-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-32-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-33-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-33-1">
Past versions of unnamed-chunk-33-1.png
</button>
</p>
<div id="fig-unnamed-chunk-33-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-33-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>outlier per sample after all filters.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-34-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-34-1">
Past versions of unnamed-chunk-34-1.png
</button>
</p>
<div id="fig-unnamed-chunk-34-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-34-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Number of outlier samples per outlier intron
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-35-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-35-1">
Past versions of unnamed-chunk-35-1.png
</button>
</p>
<div id="fig-unnamed-chunk-35-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-35-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="extreme-samples-52111" class="section level3">
<h3>extreme samples: 52111</h3>
<p>Samples with &gt; 500 outliers</p>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"52111","2":"male","3":"50","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, ~10-15% interstitial fat/vessels","7":"Gastrocnemius below patella","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"Intracranial hemmorhage","12":"Natural","13":"GTEX-ZTTD-0326-SM-57WFW"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
Samples with less than 100 outliers
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"52096","2":"male","3":"56","4":"Muscle","5":"RNA:Total RNA","6":"2  ~9x6mm aliquots, ~40% interstitial adipose tissue","7":"Below knee-Left","8":"Normal","9":"RNA isolation_Trizol Manual (Cell Pellet)","10":"Actual Death","11":"Cardiovascular","12":"Natural","13":"GTEX-QDVN-2426-SM-2S1Q4"},{"1":"52104","2":"female","3":"42","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces. 19x12 & 16.5x15mm; specimen sequenced and fingerprint confirmed as match to other donor specimens","7":"Gastrocnemius below patella","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"s/p Hanging","12":"Suicide","13":"GTEX-Y5LM-2126-SM-4VDSY"},{"1":"52117","2":"male","3":"59","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces; includes small portion of attached tendon and fat","7":"Gastrocnemius - Left","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"natural causes","12":"Natural","13":"GTEX-13FXS-0326-SM-5K7TV"},{"1":"52151","2":"female","3":"40","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, !0% interstitial fat, rep. delineated","7":"Gastrocnemius - Left","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"cardiovascular; collapse","12":"Natural","13":"GTEX-13QBU-2426-SM-5LU4L"},{"1":"52153","2":"female","3":"51","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces 9x9 & 9x5 mm; clean except for a 1 mm nodule of fibroadipose","7":"Below Knee-Right","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"ICH","12":"Natural","13":"GTEX-S4P3-1626-SM-3K2AZ"},{"1":"52155","2":"male","3":"59","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, 11x7 & 10x9mm; ~10% interstital fat, rep area delineated","7":"Gastrocnemius below patella","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"Cardiovascular","12":"Natural","13":"GTEX-Y3I4-2326-SM-4TT81"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Notice the outlier sample is not included in the open access dataset
from GTEx. One of the reasons may be that the sample did not pass QC
filters.</p>
<pre><code>character(0)</code></pre>
</div>
</div>
<div id="group2" class="section level2">
<h2>group2</h2>
<div id="intron-selection-2" class="section level3">
<h3>Intron selection</h3>
<p>An intron excision is considered outlier only if:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
</ol>
<p>Some samples have many more outlier splicing events
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-47-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-47-1">
Past versions of unnamed-chunk-47-1.png
</button>
</p>
<div id="fig-unnamed-chunk-47-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-47-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/2bbcc6698e204dd1bcf6b65aef44fa3c1296f2da/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-47-1.png" target="_blank">2bbcc66</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-47-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/22456a28518e1cf44e6565c1d6816365f0bff13d/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-47-1.png" target="_blank">22456a2</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-14
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="box-plot-before-quantile-filter-2" class="section level4">
<h4>Box plot before quantile filter</h4>
<p>There are 11776 outlier introns.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-48-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-48-1">
Past versions of unnamed-chunk-48-1.png
</button>
</p>
<div id="fig-unnamed-chunk-48-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-48-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/2bbcc6698e204dd1bcf6b65aef44fa3c1296f2da/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-48-1.png" target="_blank">2bbcc66</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/df9d86f6884184a426865d02569fbd7220f689b4/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-48-1.png" target="_blank">df9d86f</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-49-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-49-1">
Past versions of unnamed-chunk-49-1.png
</button>
</p>
<div id="fig-unnamed-chunk-49-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-49-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="further-filter-outlier-introns-by-comparing-excision-quantiles-2"
class="section level3">
<h3>Further filter outlier introns by comparing excision quantiles</h3>
<p>By definition, outlier splicing events are ones that occur only in a
few individuals. Here, I further enforced this quantile based
filter:</p>
<ul>
<li>assumes that it occurs in only up to 5% of samples,</li>
<li>thus the excision ratio of the first 95% of samples should be
largely different than that of the last 5% of samples</li>
<li>here, I will only consider the case where the first 95% of samples
have much lower excision ratio, because the complementary set always
exists for a intron cluster. But we only want to select the intron
once.</li>
<li>Specifically, an outlier intron is qualified only if the mean
excision ratio of samples in the (95, 100] percentile is more than 2x sd
away from the 95th percentile, where <code>sd</code> is estimated by
(0,95] percentiles.</li>
</ul>
<p>In addition, for any intron usage of a sample where excision ratio is
1 and observed read counts &lt; 10, read counts and excision ratio is
forced to 0. This helps remove inflated excision ratio of 1 under such
scenarios.</p>
<p>There are 1120 outlier introns after this round of quantile based
filter. The box plot below randomly select 50 out of the outlier introns
(x is sorted by p value).</p>
<p>In summary, the two box plots below plotted some of the outlier
introns that satisfy these criteria:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
<li><strong>AND</strong> quantile filter described above</li>
<li><strong>AND</strong> exclude outliers from extreme samples</li>
</ol>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-51-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-51-1">
Past versions of unnamed-chunk-51-1.png
</button>
</p>
<div id="fig-unnamed-chunk-51-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-51-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-52-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-52-1">
Past versions of unnamed-chunk-52-1.png
</button>
</p>
<div id="fig-unnamed-chunk-52-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-52-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>outlier per sample after all filters.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-53-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-53-1">
Past versions of unnamed-chunk-53-1.png
</button>
</p>
<div id="fig-unnamed-chunk-53-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-53-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Number of outlier samples per outlier intron
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-54-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-54-1">
Past versions of unnamed-chunk-54-1.png
</button>
</p>
<div id="fig-unnamed-chunk-54-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-54-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="extreme-samples-55582-55898" class="section level3">
<h3>extreme samples: 55582, 55898</h3>
<p>Samples with &gt; 500 outliers</p>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"55582","2":"male","3":"59","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces  ~12x8mm.  Intertitial fat/fibrous scar tissue is  ~10% of specimens","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Blunt Injury","12":"Accident","13":"GTEX-VUSG-2626-SM-5S2TM"},{"1":"55898","2":"male","3":"55","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces,  ~12x10mm.  ~40-50% interstitial fat, mildly atrophic-appearing skeletal muscle fibers.","7":"Below Knee-Right","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Cross Clamp Applied","11":"NA","12":"NA","13":"GTEX-TMKS-0326-SM-3DB8A"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
Samples with less than 100 outliers
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"54492","2":"male","3":"62","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces,  '10% interstitial fat,  rep. foci delineated","7":"Gastrocnemius below patella","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"alcoholic cirrhosis of liver","12":"Natural","13":"GTEX-139T8-0526-SM-5KLZL"},{"1":"54539","2":"female","3":"55","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces; 1 piece includes ~10% attached fat","7":"Gastrocnemius - Left","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"natural causes","12":"Natural","13":"GTEX-13OVJ-0226-SM-5L3GK"},{"1":"54647","2":"male","3":"62","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces 9 x 5 and 8 x 5 mm.  10% internal adipose.","7":"Gastrocnemius below patella","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Presumed Death","11":"Cardiac Arrest","12":"Natural","13":"GTEX-Y8DK-0326-SM-4TT3P"},{"1":"54650","2":"female","3":"50","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, 10x7 & 11x10mm; less than 2% interstitial fat, good specimens","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"ICH","12":"Natural","13":"GTEX-Y114-2526-SM-4TT8M"},{"1":"54657","2":"female","3":"33","4":"Muscle","5":"RNA:Total RNA","6":"2  ~8x7mm pieces, 50% fat.  ? myositis/atrophic process.  Withered muscle, fibrosis with fatty infiltration","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Cardiovascular","12":"Natural","13":"GTEX-P4PP-1626-SM-2HMJF"},{"1":"54685","2":"female","3":"56","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, but fragmented","7":"Gastrocnemius - Left","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"breathing arrested","12":"Natural","13":"GTEX-11ZVC-2726-SM-5FQSZ"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Notice the outlier sample is not included in the open access dataset
from GTEx. One of the reasons may be that the sample did not pass QC
filters.</p>
<pre><code>character(0)</code></pre>
</div>
</div>
<div id="group3" class="section level2">
<h2>group3</h2>
<div id="intron-selection-3" class="section level3">
<h3>Intron selection</h3>
<p>An intron excision is considered outlier only if:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
</ol>
<p>Some samples have many more outlier splicing events
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-66-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-66-1">
Past versions of unnamed-chunk-66-1.png
</button>
</p>
<div id="fig-unnamed-chunk-66-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-66-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="box-plot-before-quantile-filter-3" class="section level4">
<h4>Box plot before quantile filter</h4>
<p>There are 9323 outlier introns.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-67-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-67-1">
Past versions of unnamed-chunk-67-1.png
</button>
</p>
<div id="fig-unnamed-chunk-67-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-67-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-68-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-68-1">
Past versions of unnamed-chunk-68-1.png
</button>
</p>
<div id="fig-unnamed-chunk-68-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-68-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div
id="further-filter-outlier-introns-by-comparing-excision-quantiles-3"
class="section level3">
<h3>Further filter outlier introns by comparing excision quantiles</h3>
<p>By definition, outlier splicing events are ones that occur only in a
few individuals. Here, I further enforced this quantile based
filter:</p>
<ul>
<li>assumes that it occurs in only up to 5% of samples,</li>
<li>thus the excision ratio of the first 95% of samples should be
largely different than that of the last 5% of samples</li>
<li>here, I will only consider the case where the first 95% of samples
have much lower excision ratio, because the complementary set always
exists for a intron cluster. But we only want to select the intron
once.</li>
<li>Specifically, an outlier intron is qualified only if the mean
excision ratio of samples in the (95, 100] percentile is more than 2x sd
away from the 95th percentile, where <code>sd</code> is estimated by
(0,95] percentiles.</li>
</ul>
<p>In addition, for any intron usage of a sample where excision ratio is
1 and observed read counts &lt; 10, read counts and excision ratio is
forced to 0. This helps remove inflated excision ratio of 1 under such
scenarios.</p>
<p>There are 690 outlier introns after this round of quantile based
filter. The box plot below randomly select 50 out of the outlier introns
(x is sorted by p value).</p>
<p>In summary, the two box plots below plotted some of the outlier
introns that satisfy these criteria:</p>
<ol style="list-style-type: decimal">
<li>p-value of a given intron excision event &lt; 1e-07</li>
<li><strong>AND</strong> total reads of the cluster &gt; 30</li>
<li><strong>AND</strong> intron excision ratio is at least 0.05 away
from median</li>
<li><strong>AND</strong> quantile filter described above</li>
<li><strong>AND</strong> exclude outliers from extreme samples</li>
</ol>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-70-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-70-1">
Past versions of unnamed-chunk-70-1.png
</button>
</p>
<div id="fig-unnamed-chunk-70-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-70-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-71-1.png" width="960" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-71-1">
Past versions of unnamed-chunk-71-1.png
</button>
</p>
<div id="fig-unnamed-chunk-71-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-71-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>outlier per sample after all filters.</p>
<p><img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-72-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-72-1">
Past versions of unnamed-chunk-72-1.png
</button>
</p>
<div id="fig-unnamed-chunk-72-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-72-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>Number of outlier samples per outlier intron
<img src="figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-73-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-73-1">
Past versions of unnamed-chunk-73-1.png
</button>
</p>
<div id="fig-unnamed-chunk-73-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/daiuc/snaptron/blob/0f7700088861442edb0db54ba8c3b11bc16a7f0e/docs/figure/leafcutterMD_muscle_202208012.Rmd/unnamed-chunk-73-1.png" target="_blank">0f77000</a>
</td>
<td>
daiuc
</td>
<td>
2022-08-15
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="extreme-samples-58154" class="section level3">
<h3>extreme samples: 58154</h3>
<p>Samples with &gt; 500 outliers</p>
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"58154","2":"female","3":"58","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces 12x8 & 8x8 mm; minimal adipose; tissue cracking (may be histologic)","7":"Below knee-Left","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Asphyxiation","12":"Accident","13":"GTEX-S32W-2326-SM-5S2S6"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
Samples with less than 100 outliers
<div data-pagedtable="false">
<script data-pagedtable-source type="application/json">
{"columns":[{"label":["rail_id"],"name":[1],"type":["int"],"align":["right"]},{"label":["Sex"],"name":[2],"type":["chr"],"align":["left"]},{"label":["AGE"],"name":[3],"type":["int"],"align":["right"]},{"label":["Histological_Type"],"name":[4],"type":["chr"],"align":["left"]},{"label":["Analyte_Type"],"name":[5],"type":["chr"],"align":["left"]},{"label":["SMPTHNTS"],"name":[6],"type":["chr"],"align":["left"]},{"label":["SMSMPSTE"],"name":[7],"type":["chr"],"align":["left"]},{"label":["SMSTYP"],"name":[8],"type":["chr"],"align":["left"]},{"label":["SMNABTCHT"],"name":[9],"type":["chr"],"align":["left"]},{"label":["TRTPTREF"],"name":[10],"type":["chr"],"align":["left"]},{"label":["DTHFUCOD"],"name":[11],"type":["chr"],"align":["left"]},{"label":["DTHMNNR"],"name":[12],"type":["chr"],"align":["left"]},{"label":["SAMPID"],"name":[13],"type":["chr"],"align":["left"]}],"data":[{"1":"56498","2":"male","3":"64","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces; up to 10% internal fat and small portion of attached tendon","7":"Gastrocnemius - Left","8":"Normal","9":"RNA Extraction from Paxgene-derived Lysate Plate Based","10":"Actual Death","11":"cardiac arrest","12":"Natural","13":"GTEX-13JVG-0126-SM-5L3D3"},{"1":"56515","2":"male","3":"26","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, 10x8.6 & 9.4x7.9 mm","7":"Below knee-Left","8":"Normal","9":"RNA isolation_Trizol Manual (Cell Pellet)","10":"Actual Death","11":"Self Inflicted GSW to head","12":"Suicide","13":"GTEX-QESD-1626-SM-2S1RB"},{"1":"56548","2":"male","3":"68","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces,  ~ 9x7mm, areas of poor fixation","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Heart failure","12":"Natural","13":"GTEX-P4QR-0726-SM-2I5GO"},{"1":"56554","2":"female","3":"24","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces  ~11x6mm, minimal interstitial fat","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Trauma-blunt force trauma secondary to MVA","12":"Accident","13":"GTEX-T2IS-2626-SM-32QPP"},{"1":"56555","2":"male","3":"47","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces, 11x10 & 11x7mm; ~50-10% interstitial fat, rep. foci delineated","7":"Gastrocnemius below patella","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Anoxic encephalopathy","12":"Natural","13":"GTEX-XBEW-1026-SM-4QARV"},{"1":"56579","2":"male","3":"41","4":"Muscle","5":"RNA:Total RNA","6":"2 pieces  ~8x7mm,  minimal interstitial fat","7":"Below Knee-Right","8":"Normal","9":"RNA isolation_PAXgene Tissue miRNA","10":"Actual Death","11":"Blunt Injury","12":"Accident","13":"GTEX-U3ZM-1226-SM-3DB9G"}],"options":{"columns":{"min":{},"max":[10]},"rows":{"min":[10],"max":[10]},"pages":{}}}
  </script>
</div>
<p>Notice the outlier sample is not included in the open access dataset
from GTEx. One of the reasons may be that the sample did not pass QC
filters.</p>
<pre><code>character(0)</code></pre>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-conda-linux-gnu (64-bit)
Running under: Ubuntu 20.04.2 LTS

Matrix products: default
BLAS/LAPACK: /scratch/midway2/chaodai/miniconda3/envs/smk/lib/libopenblasp-r0.3.18.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] furrr_0.3.0       future_1.26.1     tictoc_1.0.1      DT_0.23          
 [5] ggrepel_0.9.1     ggrastr_1.0.1     data.table_1.14.2 forcats_0.5.1    
 [9] stringr_1.4.0     dplyr_1.0.9       purrr_0.3.4       readr_2.1.2      
[13] tidyr_1.2.0       tibble_3.1.7      ggplot2_3.3.6     tidyverse_1.3.1  

loaded via a namespace (and not attached):
 [1] fs_1.5.2          bit64_4.0.5       lubridate_1.8.0   httr_1.4.3       
 [5] rprojroot_2.0.3   tools_4.1.0       backports_1.4.1   bslib_0.3.1      
 [9] utf8_1.2.2        R6_2.5.1          vipor_0.4.5       DBI_1.1.2        
[13] colorspace_2.0-3  withr_2.5.0       tidyselect_1.1.2  bit_4.0.4        
[17] compiler_4.1.0    git2r_0.30.1      cli_3.3.0         rvest_1.0.2      
[21] xml2_1.3.3        labeling_0.4.2    sass_0.4.1        scales_1.2.0     
[25] digest_0.6.29     rmarkdown_2.14    R.utils_2.11.0    pkgconfig_2.0.3  
[29] htmltools_0.5.2   parallelly_1.32.0 dbplyr_2.2.0      fastmap_1.1.0    
[33] highr_0.9         htmlwidgets_1.5.4 rlang_1.0.2       readxl_1.4.0     
[37] rstudioapi_0.13   jquerylib_0.1.4   generics_0.1.2    farver_2.1.0     
[41] jsonlite_1.8.0    R.oo_1.24.0       magrittr_2.0.3    Rcpp_1.0.8.3     
[45] ggbeeswarm_0.6.0  munsell_0.5.0     fansi_1.0.3       lifecycle_1.0.1  
[49] R.methodsS3_1.8.1 stringi_1.7.6     whisker_0.4       yaml_2.3.5       
[53] grid_4.1.0        parallel_4.1.0    listenv_0.8.0     promises_1.2.0.1 
[57] crayon_1.5.1      haven_2.5.0       hms_1.1.1         knitr_1.39       
[61] pillar_1.7.0      codetools_0.2-18  reprex_2.0.1      glue_1.6.2       
[65] evaluate_0.15     modelr_0.1.8      vctrs_0.4.1       tzdb_0.3.0       
[69] httpuv_1.6.5      cellranger_1.1.0  gtable_0.3.0      assertthat_0.2.1 
[73] xfun_0.31         broom_0.8.0       later_1.3.0       beeswarm_0.4.0   
[77] workflowr_1.7.0   globals_0.15.0    ellipsis_0.3.2   </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
